benchmarks/cuqi-ct/test_output.py

```diff
@@ -0,0 +1,159 @@
+
+# Check basic model properties
+print('Checking model properties')
+assert model_Gaussian.get_input_sizes() == [256**2]
+assert model_GMRF.get_input_sizes() == [256**2]
+assert model_CMRF.get_input_sizes() == [256**2]
+assert model_LMRF.get_input_sizes() == [256**2]
+assert model_exactSolution.get_input_sizes() == [0]
+
+assert model_Gaussian.get_output_sizes() == [1]
+assert model_GMRF.get_output_sizes() == [1]
+assert model_CMRF.get_output_sizes() == [1]
+assert model_LMRF.get_output_sizes() == [1]
+assert model_exactSolution.get_output_sizes() == [256**2]
+
+# Check that the models support evaluation and gradient
+print('Checking model capabilities')
+assert model_Gaussian.supports_evaluate()
+assert model_GMRF.supports_evaluate()
+assert model_CMRF.supports_evaluate()
+assert model_LMRF.supports_evaluate()
+assert model_exactSolution.supports_evaluate()
+
+assert model_Gaussian.supports_gradient()
+assert model_GMRF.supports_gradient()
+assert model_CMRF.supports_gradient()
+assert not model_LMRF.supports_gradient()
+assert not model_exactSolution.supports_gradient()
+
+# Evaluate models and compare with testproblem
+print('Evaluating models')
+output_Gaussian = model_Gaussian([parameters.tolist()])[0][0]
+output_GMRF = model_GMRF([parameters.tolist()])[0][0]
+output_CMRF = model_CMRF([parameters.tolist()])[0][0]
+output_LMRF = model_LMRF([parameters.tolist()])[0][0]
+output_exactSolution = model_exactSolution([[]])[0]
+
+# Check that the output is correct
+print('Checking model output')
+BP.prior = cuqi.distribution.Gaussian(np.zeros(256**2), 0.01,
+                              geometry=BP.likelihood.geometry)
+assert output_Gaussian == pytest.approx(BP.posterior.logpdf(parameters))
+
+BP.prior = cuqi.distribution.GMRF(np.zeros(256**2), 1/(0.01),
+                                  physical_dim=2,
+                                  geometry=BP.likelihood.geometry)
+assert output_GMRF == pytest.approx(BP.posterior.logpdf(parameters))
+
+BP.prior = cuqi.distribution.Cauchy_diff(np.zeros(256**2), 0.01,
+                                         physical_dim=2,
+                                         geometry=BP.likelihood.geometry)
+assert output_CMRF == pytest.approx(BP.posterior.logpdf(parameters))
+
+BP.prior = cuqi.distribution.Laplace_diff(np.zeros(256**2), 0.1,
+                                          physical_dim=2,
+                                          geometry=BP.likelihood.geometry)
+assert output_LMRF == pytest.approx(BP.posterior.logpdf(parameters))
+
+assert np.allclose(output_exactSolution, parameters)
+
+print('Regression testing against known norm values of output')
+assert np.linalg.norm(output_Gaussian) == pytest.approx(346327.4375793076, rel=0.02)
+assert np.linalg.norm(output_GMRF) == pytest.approx(139088.07582118618, rel=0.02)
+assert np.linalg.norm(output_CMRF) == pytest.approx(466445.19477515144, rel=0.02)
+assert np.linalg.norm(output_LMRF) == pytest.approx(471311.0366245932, rel=0.02)
```

Comment:
Since you updated the default, this reg test needs to be updated.
```suggestion
assert np.linalg.norm(output_LMRF) == pytest.approx(229401.6797480647, rel=0.02)
```